src/parser/jobs/blu/modules/GeneralCDDowntime.ts

```diff
@@ -74,4 +87,8 @@ export class GeneralCDDowntime extends CooldownDowntime {
 		{cooldowns: [this.data.actions.DRAGON_FORCE]},
 		{cooldowns: [this.data.actions.ANGELS_SNACK]},
 	]
+
+	override onCompletePrepwork() {
+		this.trackedCds = this.trackedCds.filter(cdGroup => this.calculateUsageCount(cdGroup) !== 0)
```

Comment:
I feel like a cleaner solution to this need to filter the tracked cooldowns before creating the output would be to make the core file's `output` method `protected`, then override that here, run this filtering, then call `super.output()` to generate the output off the now-filtered list.

Reply:
`CooldownDowntime` doesn't use `output()` at all, since -- like with the ABC and DoT report -- they just use their `onComplete` to directly add to the checklist array.

I could make `onComplete` protected and override that? That's actually how BLU & MCH's ABC report does this already, so there's prior art for it.

Reply:
Yeah, that's still the gist of what I was getting at, even though I got the exact method name wrong. Needs more coffee before I go code reviewing!

Reply:
Alright, should be patched now!